website/docs/guides/migration/versions/06-upgrading-to-v1.2.md

```diff
@@ -27,11 +34,8 @@ See GitHub discussion [dbt-labs/dbt-core#5468](https://github.com/dbt-labs/dbt-c
 
 ## New and changed documentation
```

Comment:
nit: not sure what if "documentation" is meaningful here...
```suggestion
## New and changed functionality
```

Reply:
Fair! The original intent of this page was to link to all docs changes associated with the new version. Which is basically 1:1 with new/changed functionality. So, I'm good with either :)